test/benchmark/PhpUnitTestCaseBench.php

```diff
@@ -5,7 +5,7 @@
 namespace Roave\BetterReflectionBenchmark;
 
 use PhpBench\Benchmark\Metadata\Annotations\Iterations;
-use PHPUnit\Framework\TestCase;
+use Psalm\Config;
```

Comment:
Seems wrong: psalm is not installed, so this will likely bite us?

Reply:
Alternatively, let's look at a different file, such as a stub?

Reply:
Yeah, perhaps something in phpbench itself would be good: any cross-version comparisons are foobar anyway, at this point :D

Reply:
Change to biggest file in PhpBench itself.

Reply:
Thanks!